src/client/components/atoms/burgerButton/index.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import { useContext, useState } from 'react';
+import { Link } from 'react-router-dom';
+import StateContext from '../../../utils/stateContext';
+import ExpandMenu from '../expand';
+import './styles.scss';
+
+const BurgerButton = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const { user } = useContext(StateContext);
+
+  const handleToggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+  return (
```

Comment:
lets separate the return statement from the rest of variables/handlers/state/whatever
suggestion
`const handleToggleMenu = () => {
    setIsOpen(!isOpen);
  };

  return (...`